MvvmCross-Forms/MvvmCross.Forms/Core/MvxContentPage.cs

```diff
@@ -17,6 +17,20 @@ public IMvxViewModel ViewModel
             set { DataContext = value; }
         }
 
+        protected override void OnAppearing()
+        {
+            base.OnAppearing();
+            ViewModel?.Appearing();
```

Comment:
I'm actually not sure we need this change because the underlaying MvxActivity should trigger this.